cpp/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+use_cxx11 ()
+
```

Comment:
No. We need to keep support C++03 for a while yet.

I think this is just @jdubrule playing around with things to see what works and what doesn't.